### ISSUE TITLE
Action is broken as of 1/25/2024 by upstream shenanigans

### ISSUE THREAD

**Issue by pdehlke** [wagoid/commitlint-github-action#755]
commitlint did this this morning:

https://github.com/conventional-changelog/commitlint/pull/3871

And now this action is failing all over with

> Error: You have commit messages with errors
> 
> error running commitlint
> Found invalid rule names: header-trim. Supported rule names are: body-case, body-empty, body-full-stop, body-leading-blank, body-max-length, body-max-line-length, body-min-length, footer-empty, footer-leading-blank, footer-max-length, footer-max-line-length, footer-min-length, header-case, header-full-stop, header-max-length, header-min-length, references-empty, scope-case, scope-empty, scope-enum, scope-max-length, scope-min-length, signed-off-by, subject-case, subject-empty, subject-full-stop, subject-max-length, subject-min-length, subject-exclamation-mark, trailer-exists, type-case, type-empty, type-enum, type-max-length, type-min-length
> RangeError: Found invalid rule names: header-trim. Supported rule names are: body-case, body-empty, body-full-stop, body-leading-blank, body-max-length, body-max-line-length, body-min-length, footer-empty, footer-leading-blank, footer-max-length, footer-max-line-length, footer-min-length, header-case, header-full-stop, header-max-length, header-min-length, references-empty, scope-case, scope-empty, scope-enum, scope-max-length, scope-min-length, signed-off-by, subject-case, subject-empty, subject-full-stop, subject-max-length, subject-min-length, subject-exclamation-mark, trailer-exists, type-case, type-empty, type-enum, type-max-length, type-min-length
>     at Object.lint [as default] (/node_modules/@commitlint/lint/lib/lint.js:53:15)
>     at processTicksAndRejections (node:internal/process/task_queues:96:5)
>     at async /run.js:302:17
>     at async Promise.all (index 0)
>     at async showLintResults (/run.js:301:25)

I'm not good enough with the twisty maze that is the world of npm dependencies to understand how to fix this.

**Comment by romainbsl** [comment]
Fixing the version of commitlin to `18.5.0` as worked for me:
```yaml
    run: yarn add --no-lockfile @commitlint/config-conventional@18.5.0
```

**Comment by wagoid** [comment]
Hi @pdehlke! I couldn't replicate the issue, but I'm upgrading commitlint to latest version since @romainbsl mentioned that it has been fixed on 18.5.0. Please let me know if it still doesn't work after the upgrade 👌
